### PR TITLE
Fix fairseq version identifier in setup.py

### LINKED ISSUE #168: fairseq version 0.5.0 requirement
It seems that on 12 jun there was a major update to fairseq, changing the library structure, and Translate has already started using this. The fairseq package installed in my miniconda environment following the installation instructions is 0.4.0. Running any translate command fails with 

```
from fairseq import (
ImportError: cannot import name 'tasks'

```

I think this old version of fairseq was installed as part of the combined PyTorch and Caffe2 Conda package. pip/conda don't know about the newer version.

Could you update the installation instructions so that the current version of Translate can be run?

### PR DESCRIPTION
**Pull request by myleott**
Fixes #168

### CODE REVIEW

**Comment by jmp84** [comment]
I'm not sure this is needed since the issue may be fixed by uninstall/reinstall?

File: setup.py [hunk 1]
```diff
@@ -23,10 +23,10 @@ def requirements():
     license='BSD',
     packages=find_packages(),
     install_requires=[
-        'fairseq',
+        'fairseq>=0.5.0',
     ],
     dependency_links=[
-        'git+https://github.com/pytorch/fairseq.git#egg=fairseq',
+        'git+https://github.com/pytorch/fairseq.git#egg=fairseq-0.5.0',
```
Comment (jmp84) [inline setup.py]: @myleott, so, line 26 seems good but is this change (line 29) necessary?

Comment (myleott) [reply setup.py]: Yes, unfortunately dependency links don't get associated with version numbers unless they are explicitly added to the URI. Without this change, `setup.py install` will complain about not being able to find 0.5.0. Without line 26 it won't properly update if you have 0.4.0 :/

Btw, this should still pull the latest master commit of fairseq, the version numbers are only loosely enforced by setuptools.

**Comment by jmp84** [comment]
>Btw, this should still pull the latest master commit of fairseq, the version numbers are only loosely enforced by setuptools.

Awesome, that's what I was worried about.

**Pull request closed without merging**